hm_pyhelper/hardware_definitions.py

```diff
@@ -129,7 +137,7 @@ def is_raspberry_pi() -> bool:
         },
 
     # Nebra Radxa 0 Light Hotspot SPI Ethernet
-    'NEBHNT-LGT-RADXA': {
+    'nebra-light2': {
```

Comment:
Nebra Light Hotspot II has two configuration options: Both Raspi Zero and Radxa Zero. Do we ignore the former and go directly with Radxa one only?

Reply:
@MuratUrsavas this is taken care of in https://github.com/NebraLtd/hm-pyhelper/pull/173